lib/xapi/track.rb

```diff
@@ -86,6 +87,11 @@ def doc_format
       most_popular_format(path) || default_format
     end
 
+    def global_zip
+      global_dir = dir.join("global")
+      @zip ||= FileBundle.create_zip(global_dir, FileBundle.paths(global_dir))
```

Comment:
Whenever I see something like this, where the argument to a method is the result of another call to the same class with the same variable:
`FileBundle.create_zip(global_dir, FileBundle.paths(global_dir))`
I wonder, why cant `create_zip` just work out the paths itself?
It would be much nicer for us as the caller to be able to do:
`FileBundle.create_zip(global_dir)`
and let FileBundle worry about working out the paths itself.